packages/run-protocol/src/vaultFactory/AttackersGuide.md

```diff
@@ -0,0 +1,75 @@
+# An Attacker's guide to the Vault Factory
+
+This is an incomplete list of potential weak points that an attacker might want to focus
+on when looking for ways to violate the integrity of the Vault Factory. It's here to help
+defenders, as "attacker's mindset" is a good way to focus attention for the defender. The
+list should correspond pretty closely to the set of assurances that the Vault Factory aims
+to support.
+
+The VaultFactory's purpose is to make it possible for users to deposit hard currencies and
+borrow RUN in exchange. They will pay interest on the total outstanding balance, and have
+to pay off the debt or deposit additional collateral to ensure that the ratio of debt to
+collateral stays above a threshold. If the ratio falls below the limit, their collateral
+will be sold.
+
+Part of the purpose of the Vault mechanism is to link the price of RUN to
+dollar-equivalents. We're currently getting prices from the PriceAuthority in RUN, but
+this should (?) be changed to USDC or equivalent. Tying the price at which Vaults lend RUN
```

Comment:
please point to an issue about this "should be changed" stuff.

Reply:
https://github.com/Agoric/agoric-sdk/issues/4714